lib/adapter.js

```diff
@@ -100,9 +100,19 @@ var objectTrackingCallbackInterval =
 // Metrics API Use function
 function use(callback) {
 
+  exports.on('watchdogActivationCount', function(count) {
+    callback('watchdog.activations.count', count);
+  });
+
   // The callback needs to fire every 60 seconds with data from the last 60
   // seconds
   setInterval(function() {
+
+    if (appmetrics && appmetrics.watchdogActivationCount) {
+      exports.emit('watchdogActivationCount',
```

Comment:
Strong agent does this emit in the 'poll' function ([https://github.com/strongloop/strong-agent/blob/master/lib/agent.js#L299-L302](https://github.com/strongloop/strong-agent/blob/master/lib/agent.js#L299-L302)). I put it here because it seemed like this callback that runs every 60 seconds was our equivalent of poll. Happy to move it if there's a better place though.